packages/codegen/src/codegen/.hygen/templates/powerhouse/generate-document-model-module/exceptions.esm.t

```diff
@@ -0,0 +1,30 @@
+---
+to: "<%= rootDir %>/<%= h.changeCase.param(documentType) %>/gen/<%= module %>/exceptions.ts"
+force: true
+---
+
+<% actions.forEach(action => { _%>
+<% action.errors.forEach(error => { _%>
+class <%= h.changeCase.pascal(error.name) %> extends Error {
+    errorCode = '<%= h.changeCase.pascal(error.name) %>';
+    constructor(message = '<%= h.changeCase.pascal(error.name) %>') {
+        super(message);
+    }
+}
+
+<% }); _%>
+<% }); _%>
+
+
+export default {
```

Comment:
Let's avoid default exports and use named exports instead.

Exporting the error classes is interesting but I think we should also export each error an object and type/interface for each error, since classes are for runtime only.

Reply:
fix: [fe3acc8](https://github.com/powerhouse-inc/powerhouse/pull/499/commits/fe3acc8cc9ed6c6420ba92eb644fd38a01fbcbb5)